src/app/prevention/prevention.component.scss

```diff
@@ -0,0 +1,5 @@
+@import "../styles/main";
+
+.content {
+  margin-bottom: 100px;
```

Comment:
check if 100 px is already defined as a variable.. if not define it in the scss structure and then use.